include/microcdr/common.h

```diff
@@ -55,14 +41,43 @@ typedef struct MicroBuffer
     Endianness endianness;
     uint32_t last_data_size;
 
-    BufferError error;
+    bool error;
```

Comment:
Create a function to get the error flag and another to unset the error flag

Reply:
Done!